site/reference/live/apply/README.md

```diff
@@ -1,298 +1,36 @@
-retry forever, so this should not be considered a terminal state.
-
-These conditions adhere to the [Kubernetes design principles]
-which include expressing conditions using abnormal-true polarity. There is
-currently a [proposal] to change to guidance for conditions. If this is
-accepted, the recommended conditions for kpt might also change, but we will
-continue to support the current set of conditions.
-
-CRDs should also set the `observedGeneration` field in the status object, a
-pattern already common in the built-in types. The controller should update
-this field every time it sees a new generation of the resource. This allows
-the kpt library to distinguish between resources that do not have any
-conditions set because they are fully reconciled, from resources that have no
-conditions set because they have just been created.
-
-An example of a resource where the latest change has been observed by
-the controller which is currently in the process of reconciling would be:
-
-```yaml
-apiVersion: example.com
-kind: Foo
-metadata:
-  generation: 12
-  name: bar
-spec:
-  replicas: 1
-status:
-  observedGeneration: 12
-  conditions:
-  - lastTransitionTime: "2020-03-25T21:20:38Z"
-    lastUpdateTime: "2020-03-25T21:20:38Z"
-    message: Resource is reconciling
-    reason: Reconciling
-    status: "True"
-    type: Reconciling
-  - lastTransitionTime: "2020-03-25T21:20:27Z"
-    lastUpdateTime: "2020-03-25T21:20:39Z"
-    status: "False"
-    type: Stalled
-```
-
-The status for this resource state will be InProgress. So if the
-`--reconcile-timeout` flag is set, kpt live apply will wait until
-the `Reconciling` condition is `False` before pruning and exiting.
-
-### Examples
-
-{{% hide %}}
-
-<!-- @makeWorkplace @verifyExamples-->
-```
-# Set up workspace for the test.
-TEST_HOME=$(mktemp -d)
-cd $TEST_HOME
-```
-
-<!-- @fetchPackage @verifyExamples-->
-```shell
-export SRC_REPO=https://github.com/GoogleContainerTools/kpt.git
-kpt pkg get $SRC_REPO/package-examples/helloworld-set@next my-dir
-```
-
-<!-- @createKindCluster @verifyExamples-->
-```
-kind delete cluster && kind create cluster
-```
-
-<!-- @installResourceGroup @verifyExamples-->
-```
-kpt live install-resource-group
-```
-
-<!-- @initCluster @verifyExamples-->
-```
-kpt live init my-dir
-```
-
-{{% /hide %}}
-
-<!--mdtogo:Examples-->
-<!-- @liveApply @verifyExamples-->
-```shell
-# apply resources and prune
-kpt live apply my-dir/
-```
-
-<!-- @liveApply @verifyExamples-->
-```shell
-# apply resources and wait for all the resources to be reconciled before pruning
-kpt live apply --reconcile-timeout=15m my-dir/
-```
-
-<!-- @liveApply @verifyExamples-->
-```shell
-# apply resources and specify how often to poll the cluster for resource status
-kpt live apply --reconcile-timeout=15m --poll-period=5s my-dir/
-```
-<!--mdtogo-->
-
 ### Synopsis
 <!--mdtogo:Long-->
 ```
-kpt live apply DIR [flags]
+kpt live apply [DIR|-] [flags]
```

Comment:
We're using `PKG_PATH` for package directories. `DIR` for normal directories. e.g.:

https://kpt.dev/reference/pkg/update/

Same for other commands.

Reply:
We should also have a consistent description for the flag. "Path to a directory containing KRM resources and a Kptfile" is a awkward (The critical thing is Kptfile, otherwise not having any other KRM resources is a valid case).

Reply:
I have updated the reference docs for all the live commands. I added some new language to the description, but I'm open to suggestions here.